stdlib/importlib/resources/__init__.pyi

```diff
@@ -1,37 +1,67 @@
 import os
 import sys
+from _typeshed import StrPath
 from collections.abc import Iterator
 from contextlib import AbstractContextManager
+from io import TextIOWrapper
 from pathlib import Path
 from types import ModuleType
-from typing import Any, BinaryIO, TextIO
-from typing_extensions import TypeAlias
+from typing import Any, BinaryIO, TextIO, overload
+from typing_extensions import TypeAlias, Unpack
 
 if sys.version_info >= (3, 9):
     from importlib.abc import Traversable
 
-__all__ = ["Package", "Resource", "contents", "is_resource", "open_binary", "open_text", "path", "read_binary", "read_text"]
+__all__ = ["Package", "contents", "is_resource", "open_binary", "open_text", "path", "read_binary", "read_text"]
 
 if sys.version_info >= (3, 9):
     __all__ += ["as_file", "files"]
 
 if sys.version_info >= (3, 10):
     __all__ += ["ResourceReader"]
 
+if sys.version_info >= (3, 13):
+    __all__ += ["Anchor"]
+else:
+    __all__ += ["Resource"]
+
 Package: TypeAlias = str | ModuleType
 
-if sys.version_info >= (3, 11):
-    Resource: TypeAlias = str
-else:
+if sys.version_info >= (3, 13):
+    Anchor: TypeAlias = Package
```

Comment:
I'd put these into `._common`, like in the implementation (introduced in Python 3.11). Or at least `Anchor`, since it was newly introduced.

Reply:
Fixed! https://github.com/python/typeshed/pull/12298/commits/9cc45859783d25ec81d1b741045e18742bb4b4d7